--- boards/evkmimxrt1170/demo_apps/avb_tsn/common/board.h_old	2021-12-15 14:39:19.033724405 +0100
+++ boards/evkmimxrt1170/demo_apps/avb_tsn/common/board.h	2021-12-15 14:39:34.098252438 +0100
@@ -113,7 +113,7 @@
 #define BOARD_NUM_PORTS      1
 #define BOARD_MAX_PORTS      2
 
-#define BOARD_REV_B          0
+#define BOARD_REV_B          1
 
 #if BOARD_NUM_PORTS > BOARD_MAX_PORTS
 #error Invalid number of ports
