From 1d8aa8a3ae6bf8c3204b26d2760b3c4ca0c02195 Mon Sep 17 00:00:00 2001
From: Nicolas Diaz <nicolas.diaz@nxp.com>
Date: Fri, 3 Dec 2021 13:21:38 +0100
Subject: [PATCH] Use-Basler-device-tree-for-Linux

Signed-off-by: Nicolas Diaz <nicolas.diaz@nxp.com>
---
 include/configs/imx8mp_evk.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/imx8mp_evk.h b/include/configs/imx8mp_evk.h
index 14866b1796..b45dfa413e 100644
--- a/include/configs/imx8mp_evk.h
+++ b/include/configs/imx8mp_evk.h
@@ -211,7 +211,7 @@
 	"boot_fdt=try\0" \
 	"fdt_high=0xffffffffffffffff\0"		\
 	"boot_fit=no\0" \
-	"fdtfile=" CONFIG_DEFAULT_FDT_FILE "\0" \
+	"fdtfile=imx8mp-evk-basler-ov5640.dtb\0" \
 	"bootm_size=0x10000000\0" \
 	"mmcdev="__stringify(CONFIG_SYS_MMC_ENV_DEV)"\0" \
 	"mmcpart=" __stringify(CONFIG_SYS_MMC_IMG_LOAD_PART) "\0" \
-- 
2.17.1

